src/js/extension-anchor.js

```diff
@@ -24,6 +24,10 @@ var AnchorExtension;
 
         // Button and Extension handling
 
+        // labels for the anchor-edit form buttons
+        formSaveLabel: '&#10003;',
+        formCloseLabel: '&times;',
```

Comment:
Yeah, you're right, figuring out how to do the fontawesome labels is a bigger issue than just this, just wanted to get your thoughts.